cohesity/resource_cohesity_cloud_edition_cluster.go

```diff
@@ -275,7 +275,7 @@ func resourceCohesityCloudEditionClusterUpdate(resourceData *schema.ResourceData
 			log.Printf(err.Error())
 			return errors.New("Failed to update cloud edition cluster")
 		}
-		resourceData.SetPartial("license_key")
+		// resourceData.SetPartial("license_key")
```

Comment:
remove

Reply:
leaving it like that because there may be a need  to change it